### ISSUE TITLE
Deploy Error: None of the available transfer modes work.

### ISSUE THREAD

**Issue by SsadiqRoy** [SamKirkland/FTP-Deploy-Action#332]
**Bug Description**
I have used the FTP-Deploy-Action on other projects but is does not work this type and it does not show any real error log.
my workflow is below

**My Action Config**
```yaml
on:
  push:
    branches:
      - new-feature

name: 🚀 Deploy website on push
jobs:
  web-deploy:
    name: 🎉 Deploy
    runs-on: ubuntu-latest
    steps:
      - name: 🚚 Get latest code
        uses: actions/checkout@v2.5.0

      - name: 📂 Sync files
        uses: SamKirkland/FTP-Deploy-Action@4.3.2
        with:
          server: ftp.server.com
          username: ${{ secrets.ftp_username }}
          password: ${{ secrets.ftp_password }}
          server-dir: /public_html/new-feature.folder/
```

**My Action Log**
```
  
----------------------------------------------------------------
--------------  🔥🔥🔥 an error occurred  🔥🔥🔥  --------------
----------------------------------------------------------------

----------------------------------------------------------------
----------------------  full error below  ----------------------
----------------------------------------------------------------

Error: None of the available transfer modes work.
    at Client.prepareTransfer (/home/runner/work/_actions/SamKirkland/FTP-Deploy-Action/4.3.2/dist/index.js:4803:19)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
Error: Error: None of the available transfer modes work.
```

**Comment by SsadiqRoy** [comment]
@SamKirkland Please is there some wrong 👆

**Comment by SamKirkland** [comment]
This should be fixed in `v4.3.4`
